explainability/explainability-core/src/main/java/org/kie/kogito/explainability/local/lime/LimeExplainer.java

```diff
@@ -179,13 +180,15 @@ private void getSaliency(List<Feature> linearizedTargetInputFeatures, Map<String
         List<FeatureImportance> featureImportanceList = new LinkedList<>();
 
         // encode the training data so that it can be fed into the linear model
+        EncodingParams encodingParams = new EncodingParams(limeConfig.getEncodingGaussianFilterWidth(), limeConfig.getEncodingClusterThreshold());
         DatasetEncoder datasetEncoder = new DatasetEncoder(limeInputs.getPerturbedInputs(),
-                                                           limeInputs.getPerturbedOutputs(),
-                                                           linearizedTargetInputFeatures, originalOutput);
+                limeInputs.getPerturbedOutputs(),
+                linearizedTargetInputFeatures, originalOutput, encodingParams);
```

Comment:
What about add a `getEncodingParams()` method to `LimeConfig` so here the code become
```suggestion
                linearizedTargetInputFeatures, originalOutput, 
                limeConfig.getEncodingParams());
```